RevenueCatUI/Views/RemoteImage.swift

```diff
@@ -8,57 +8,64 @@
 //      https://opensource.org/licenses/MIT
 //
 //  RemoteImage.swift
-//  
+//
 //  Created by Nacho Soto on 7/19/23.
 
 import SwiftUI
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 struct RemoteImage: View {
 
+    @StateObject private var loader: ImageLoader
+    private let cache: URLCache
```

Comment:
Small thing but do we need to store this here? Seems like this is unused.

Reply:
good catch